Iteration 1 Complete

joshuafajardo opened the pull request:
Wanted to include more numbers (e.g. increased ___ by X%), but felt like there weren't really any experiences/projects where it would fit. Could potentially add another hackathon (?) (competed, made progress, but ultimately didn't complete project). Thoughts on projects?

joshuafajardo committed:
small edits

joshuafajardo committed:
All but projects

joshuafajardo committed:
Added Heal-Bot project

joshuafajardo committed:
Draft finished, need revision

joshuafajardo committed:
slight wording tweaks

joshuafajardo committed:
Update resume.md

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,48 @@
-* ...
+**Undergraduate Researcher, Software/Hardware**: Embodied Dexterity Group| Berkeley, CA ... *November 2019 - Present*
+
+* Led team of 3 in designing a physics engine and GUI in Python to interface with a prosthetic arm. Incorporated the Model/View/Controller (MVC) design pattern to organize functions and methods.
```
Be specific - what types of functions and methods?

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,48 @@
+**Undergraduate Researcher, Software/Hardware**: Embodied Dexterity Group| Berkeley, CA ... *November 2019 - Present*
+
+* Led team of 3 in designing a physics engine and GUI in Python to interface with a prosthetic arm. Incorporated the Model/View/Controller (MVC) design pattern to organize functions and methods.
+* Established TCP connection with LabVIEW program using Python's socket library to ensure the simulation data was sufficiently accurate for use in research.
```
how accurate?  Tell me more about how TCP Connection was critical to your project's success

alextakahashi commented on resume.md:
```diff
@@ -1,49 +1,48 @@
+
+* Led team of 3 in designing a physics engine and GUI in Python to interface with a prosthetic arm. Incorporated the Model/View/Controller (MVC) design pattern to organize functions and methods.
+* Established TCP connection with LabVIEW program using Python's socket library to ensure the simulation data was sufficiently accurate for use in research.
+* Utilized multithreading to run communication/physics and GUI processing in parallel, allowing the GUI to refresh at a consistent framerate.
```
why does this specific use case need to be multithreaded?  What were the challenges that you overcame

joshuafajardo committed:
Iteration 1.5

Made changes according to Alex's recommendations; going to update other sections correspondingly.

joshuafajardo committed:
Iteration 2 Complete
